vmck/jobs.py

```diff
@@ -43,22 +43,16 @@ def poll(job):
 
     elif status == 'running':
         job.state = job.STATE_RUNNING
-        done = nomad.cat(nomad_id(job), f'alloc/data/done', binary=True)
-        if done is not None:
-            on_done(job)
-            kill(job)
-
-        else:
-            health = nomad.health(job.id)
-            if health:
-                check = health[0]
-                if check['Status'] == 'passing':
-                    ssh_remote = {
-                        'host': check['Output'].split(':')[0].split()[-1],
-                        'port': int(check['Output'].split(':')[1]),
-                        'username': settings.SSH_USERNAME,
-                    }
-                    return ssh_remote
+        health = nomad.health(job.id)
+        if health:
+            check = health[0]
+            if check['Status'] == 'passing':
+                ssh_remote = {
+                    'host': check['Output'].split(':')[0].split()[-1],
```

Comment:
This looks complex. Maybe a comment explaining what you expect to see in `check['Output']` and where it comes from?

Reply:
If github reviews were a bit smarter, they would tell you this whole block was dedented, because I deleted the `else`.

Reply:
But yeah the code could use more clarity. I'll push a cleanup patch before I merge.